sdk/storage/storage-file-share/README.md

```diff
@@ -78,8 +78,9 @@ There are differences between Node.js and browsers runtime. When getting started
 
 ### NPM
 
-The preferred way to install the Azure File Storage client library for JavaScript is to use the npm package manager. Simply type the following into a terminal window:
+> Note: This package was previously published under the name `@azure/storage-file`. It has been renamed to `@azure/storage-file-share` which is not published to the npm yet. Please make use of the `@azure/storage-file` package until `@azure/storage-file-share` is published to npm.
```

Comment:
Will this notice be removed before GA release?

Reply:
storage-file-share is not going GA, instead we will be releasing a preview. On the day of the release, we will remove the note.

Reply:
#5935 logged an issue for it too :)